README.md

```diff
@@ -92,3 +92,23 @@ the same directory (either top level or under a series of subdirectories called
 archive. On Windows, DMLC is sometimes unable to resolve these symlinks
 correctly; for this reason, it is recommended that the archive is only
 extracted and compiled on Linux.
+
+### DMLC_GATHER_SIZE_STATISTICS
+When set, DMLC emits a `-size-stats.json` file, which outputs a file ending
+with `-size-stats.json`, which shows code generation statistics useful to
+reduce generated code size and increase compile speed. The file lists how much
+C code is generated for each DML method. The output is a list of triples
+`[tot_size, location, num]`, where `tot_size` is the total C code generated
+from one method declarations, `num` is how many times C code was generated from
+this declaration (because it was expanded by a template), and `location` is the
+source location of the declaration.
+
+An entry with a large `tot_size` and a large `num` can be by declaring the
```

Comment:
missing verb after "be" ("be by")

Likely reformat to "If you have an entry with a large 'tot_size' and a large 'num', the 'tot_size' can be reduced by..."